### PR TITLE
Fix storybook deployment

### PR DESCRIPTION
**Pull request by tblivet**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Fix storybook deployment
| Type?             | bug fix
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | https://github.com/PrestaShop/prestashop-ui-kit/issues/235
| Sponsor company   | @PrestaShopCorp
| How to test?      | -

### CODE REVIEW

**Review by kpodemski — APPROVED** [review]

**Comment by ps-jarvis** [comment]
PR merged, well done!

Message to @PrestaShop/committers: do not forget to milestone it before the merge.

**Pull request merged**
